### PR TITLE
windows/date: add Indonesian translation

### PR DESCRIPTION
**Pull request by hamexhanif**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages.id/windows/date.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# date
+
+> Menampilkan atau mengatur tanggal sistem.
+> Informasi lebih lanjut: <https://learn.microsoft.com/windows-server/administration/windows-commands/date>.
+
+- Menampilkan tanggal sistem saat ini dan meminta untuk memasukkan tanggal baru (biarkan kosong agar tidak berubah):
+
+`date`
+
+- Menampilkan tanggal sistem saat ini tanpa meminta untuk memasukkan tanggal baru:
```
Comment (reinhart1010) [inline pages.id/windows/date.md:10]:
```suggestion
- Tampilkan tanggal sistem saat ini tanpa meminta untuk memasukkan tanggal baru:
```

File: pages.id/windows/date.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+`date /t`
+
+- Mengubah tanggal sistem saat ini ke tanggal tertentu:
```
Comment (reinhart1010) [inline pages.id/windows/date.md:14]:
```suggestion
- Ubah tanggal sistem saat ini ke tanggal tertentu:
```

**Review by reinhart1010 — CHANGES_REQUESTED** [review]
Thanks, here are some of my notes to review.

**Review by kbdharun — APPROVED** [review]
Thanks for your contribution.
LGTM, after @reinhart1010's suggestions are addressed.

**Review by reinhart1010 — APPROVED** [review]
Thanks!

**Comment by kbdharun** [comment]
Note: The reason why I closed and reopened the PR was to trigger a rerun of the stuck CI check (which happens sometimes due to some issue with GitHub API). Now it has passed,  will merge the PR.

**Pull request merged**